spanner/sessionclient.go

```diff
@@ -313,6 +313,62 @@ func (sc *sessionClient) executeBatchCreateSessions(client *vkit.Client, createC
 	}
 }
 
+func (sc *sessionClient) executeCreateMultiplexedSessions(ctx context.Context, client *vkit.Client, labels map[string]string, md metadata.MD, consumer sessionConsumer) {
+	ctx = trace.StartSpan(ctx, "cloud.google.com/go/spanner.CreateSession")
+	defer func() { trace.EndSpan(ctx, nil) }()
+	trace.TracePrintf(ctx, nil, "Creating a multiplexes sessions")
+	sc.mu.Lock()
+	closed := sc.closed
+	sc.mu.Unlock()
+	if closed {
+		err := spannerErrorf(codes.Canceled, "Session client closed")
+		trace.TracePrintf(ctx, nil, "Session client closed while creating multiplexed sessions: %v", err)
+		return
+	}
+	if ctx.Err() != nil {
+		trace.TracePrintf(ctx, nil, "Context error while creating a multiplexed session: %v", ctx.Err())
+		consumer.sessionCreationFailed(ToSpannerError(ctx.Err()), 1, true)
+		return
+	}
+	var mdForGFELatency metadata.MD
+	response, err := client.CreateSession(contextWithOutgoingMetadata(ctx, sc.md, sc.disableRouteToLeader), &sppb.CreateSessionRequest{
+		Database: sc.database,
+		// Multiplexed sessions do not support labels.
```

Comment:
Either remove the labels from the input arguments of this method, or combine this method with the `createSession(..)` method and only conditionally add labels in the combined method.

Reply:
Removed the labels